src/fmu/dataio/providers/objectdata/_xtgeo.py

```diff
@@ -6,8 +6,6 @@
 from typing import TYPE_CHECKING, Final
 
 import numpy as np
-import pandas as pd
-import xtgeo
```

Comment:
This is a nice save, in theory. Both pandas and xtgeo can be slow to import (xtgeo also imports pandas), so whenever we can constraint them to being imported under a `TYPE_CHECKING` guard it's a good thing. You see this mostly during "start-up", just before execution is beginning, but with Python this can be quite noticeable (10s of seconds).

A big speed impediment to import time is Python compiling the interpreted code into byte code. Usually this means that the _first_ import is slow, and the rest of the imports are "cached" in the sense that things being located and compiled, but whenever we can reduce the dependency graph we can make that compilation faster.